packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2488,7 +2488,56 @@ void main() {
     expect(widths.indexOf(largestWidth), equals(1));
   }, skip: isBrowser); // https://github.com/flutter/flutter/issues/39998
 
-  // Regression test for https://github.com/flutter/flutter/issues/161773
```

Comment:
Could you please add this comment back to just above the `'CupertinoDatePicker date value baseline alignment'` test?